docs/src/main/asciidoc/smallrye-metrics.adoc

```diff
@@ -3,77 +3,75 @@ This guide is maintained in the main Quarkus repository
 and pull requests should be submitted there:
 https://github.com/quarkusio/quarkus/tree/main/docs/src/main/asciidoc
 ////
+
 = SmallRye Metrics
 
 include::./attributes.adoc[]
 
-This guide demonstrates how your Quarkus application can use https://github.com/smallrye/smallrye-metrics/[SmallRye Metrics],
-an implementation of the https://github.com/eclipse/microprofile-metrics/[MicroProfile Metrics] specification.
-
-SmallRye Metrics allows applications to gather various metrics and statistics that provide
-insights into what is happening inside the application.
+The following guide demonstrates how a Quarkus application can use link:https://github.com/smallrye/smallrye-metrics/[SmallRye Metrics],
+an implementation of the link:https://github.com/eclipse/microprofile-metrics/[MicroProfile Metrics] specification.
 
-The metrics can be read remotely using JSON format or the OpenMetrics format, so that
-they can be processed by additional tools such as Prometheus, and stored for analysis
-and visualisation.
+SmallRye Metrics allows applications to gather metrics and statistics that provide insights into what is happening inside an application. The metrics can be read remotely using the JSON or OpenMetrics format to be processed by additional tools such as Prometheus and stored for analysis and visualization.
 
-Apart from application-specific metrics, which are described in this guide, you may also utilize built-in metrics
-exposed by various Quarkus extensions. These are described in the guide for each particular extension that supports
-built-in metrics.
+Apart from application-specific metrics described in this guide, you may also use built-in metrics exposed by various Quarkus extensions. These are described in the guide for each particular extension that supports built-in metrics.
 
-IMPORTANT: link:micrometer[Micrometer] is the recommended approach to metrics for Quarkus.
-Use the SmallRye Metrics extension when it's required to retain MicroProfile specification compatibility.
+IMPORTANT: link:https://quarkus.io/guides/micrometer[Micrometer] is the recommended approach to metrics for Quarkus. Use the SmallRye Metrics extension when it is required to retain MicroProfile specification compatibility.
 
-== Prerequisites
+.Prerequisites
 
-To complete this guide, you need:
+* Approximately 15 minutes.
+* An IDE.
+* JDK 11+ installed with `JAVA_HOME` configured appropriately. This could be tested using the `java -version` and `javac` commands, after the execution of which a user with correctly setup Java obtains information about the current version and the possible `javac` options of usage.
```

Comment:
I don't think we need to explain this to users. A user who has some experience with Java, which is assumed here, knows what the `java` and `javac` commands do.